### DISCUSSION TITLE
Is there a way to change ListView spacing without changing the MDXT style?

### DISCUSSION THREAD

**Discussion by CrackAndDie** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#2782] [Q&A] [answered]
I have a ListView in my program with MaterialDesignXamlToolkit style set. By the problem is that spacing between items in the ListView is quite big: 
![gaps](https://user-images.githubusercontent.com/52558686/179282332-1f3467e0-2a2c-4e75-98ff-98ff95191875.png)  
I know how to change that via *ItemContainerStyle*, but it rewrites the MDXT ListView style.  
So is there a way to change the spacing without rewriting the MDXT style?

**Comment by ElieTaillard** [comment, marked as answer]
Have you tried putting negative margins on items ?

**Reply by CrackAndDie** [reply]
this helped, I forgot that I can use negative margin :)
